### PR TITLE
Modify contract test command to use sc-meta

### PR DESCRIPTION
**Pull request by popenta**
**mxpy** does not take care of `vmtools` anymore. The `vmtools` dependency has been removed.
Now, **mxpy** simply forwards the arguments to `sc-meta`. When calling `mxpy contract test`, the `sc-meta test` command is invoked. One could also use directly `sc-meta test` since `sc-meta` is installed when calling `mxpy deps install rust`.

The parameters have been changed to match the parameters needed by `sc-meta`.

`mxpy contract test` now uses the following parameters:
```
--path PATH  the directory of the contract (default: cwd)
--go         this arg runs rust and go tests (default: false)
--scen       this arg runs scenarios (default: false). If `--scen` and `--go` are both specified, scen overrides the go argument
--nocapture  this arg prints the entire output of the vm (default: false)
```

### CODE REVIEW

File: multiversx_sdk_cli/dependencies/modules.py [hunk 1]
```diff
@@ -359,6 +310,17 @@ def _install_twiggy(self):
 
         myprocess.run_process(args, env=self.get_cargo_env())
 
+    def _install_sc_meta_deps(self):
+        # this is needed for sc-meta to run the tests
+        # if os is Windows skip insallation
```
Comment (andreibancioiu) [inline multiversx_sdk_cli/dependencies/modules.py:315]: Typo.

File: multiversx_sdk_cli/dependencies/modules.py [hunk 2]
```diff
@@ -359,6 +310,17 @@ def _install_twiggy(self):
...
+        os = platform.system().lower()
+        if os == "windows":
+            return
```
Comment (andreibancioiu) [inline multiversx_sdk_cli/dependencies/modules.py:318]: Perhaps we should display a WARN?

**Review by danielailie — APPROVED** [review]

File: multiversx_sdk_cli/dependencies/modules.py (hunk 1 above)
Comment (popenta) [reply multiversx_sdk_cli/dependencies/modules.py:315]: fixed

File: multiversx_sdk_cli/dependencies/modules.py (hunk 2 above)
Comment (popenta) [reply multiversx_sdk_cli/dependencies/modules.py:319]: added a warning

**Review by andreibancioiu — APPROVED** [review]

**Review by danielailie — APPROVED** [review]

**Pull request merged**